apps/src/templates/instructions/TeacherFeedback.jsx

```diff
@@ -109,57 +111,30 @@ export class TeacherFeedback extends Component {
     this.onRubricChange = this.onRubricChange.bind(this);
 
     this.state = {
-      comment: '',
-      performance: null,
+      comment:
```

Comment:
mapping a component's props to state like this is usually an anti-pattern in react, but, after reading [this blog post](https://medium.com/@justintulk/react-anti-patterns-props-in-initial-state-28687846cc2e), it seems okay to do in this context.

i usually get around this by moving the handlers that change state for these props into the parent like this:
```
class Parent extends React.Component {
  state = {
    comment: ''
  }

  onChangeComment = (event) => {
    this.setState({comment: event.target.value});
  }

  render() {
    return <Child comment={this.state.comment} onChangeComment={this.onChangeComment} />;
  }
}

class Child extends React.Component {
  render() {
    return (
      <input value={this.props.comment} onChange={this.props.onChangeComment} />
    );
  }
}
```

then, the parent component fully controls the state and the child is basically just rendering and calling methods that control state on the parent